iml-gui/crate/src/components/table.rs

```diff
@@ -0,0 +1,61 @@
+use crate::generated::css_classes::C;
+use seed::{prelude::*, Attrs, *};
+
+pub fn wrapper_cls() -> Attrs {
+    class![C.table_auto, C.w_full]
```

Comment:
This is almost the same as defining a custom class like
```css
.wrapper-cls {
  @apply table-auto w-full;
}
```
It is not "atomic", but "molecular" for sure :)
Putting it in a CSS file might have a small advantage for not requiring recompilation Rust, by the way.